### PR TITLE
Added id parameter for getUser req

### PR DESCRIPTION
**Pull request by kasperslu**

### CODE REVIEW

File: lib/ApiClient.js [hunk 1]
```diff
@@ -9,15 +9,19 @@ class ApiClient extends AxiosClient {
     autobind(this);
   }
 
-  getUser(jwtToken, params = {}) {
-    const path = 'account';
```
Comment (aliaksandr-savitski) [inline lib/ApiClient.js]: We could set the path parameter right here, so we wouldn't need those `if {} else {}`. Like:
`const path = id ? 'drops/${id}' : 'account';`
😉 
@rafalswietek what do you think?

**Review by aliaksandr-savitski — CHANGES_REQUESTED** [review]

File: lib/ApiClient.js (hunk 1 above)
Comment (Necromos) [reply lib/ApiClient.js]: @aliaksandr-savitski Generally speaking I'm against the changes proposed here.
This function right now is getting user with specified jwtToken and passed params so adding additional param may be nice way to improve the functionality but order of params is being inconsistent with rest of the functions.
To keep proper order it should be `getUser(id = null, jwtToken, params = {})` and by doing so we will make it backward incompatible.

My suggestion is to introduce new lexical method `getUserById` with above setup of params and it should do the trick :)

**Review by Necromos — CHANGES_REQUESTED** [review]

File: lib/ApiClient.js (hunk 1 above)
Comment (kasperslu) [reply lib/ApiClient.js]: Guys, regarding @Necromos opinion, I will apply this solution. Let me know If it suits you :)

Comment (rafalswietek) [reply lib/ApiClient.js]: I've been thinking about this solution too, but it seemed to me like a non consistent method name according to the rest of this SDK.
Although I've seen worst in aws-sdk so I'm ok with it 👍

**Review by Necromos — APPROVED** [review]

**Comment by aliaksandr-savitski** [comment]
Fine with me 😉

**Review by aliaksandr-savitski — APPROVED** [review]

**Pull request merged**